Jellyfin.Api/Auth/DefaultAuthorizationPolicy/DefaultAuthorizationHandler.cs

```diff
@@ -4,41 +4,39 @@
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Http;
 
-namespace Jellyfin.Api.Auth.DefaultAuthorizationPolicy
+namespace Jellyfin.Api.Auth.DefaultAuthorizationPolicy;
+
+/// <summary>
+/// Default authorization handler.
+/// </summary>
+public class DefaultAuthorizationHandler : BaseAuthorizationHandler<DefaultAuthorizationRequirement>
 {
     /// <summary>
-    /// Default authorization handler.
+    /// Initializes a new instance of the <see cref="DefaultAuthorizationHandler"/> class.
     /// </summary>
-    public class DefaultAuthorizationHandler : BaseAuthorizationHandler<DefaultAuthorizationRequirement>
+    /// <param name="userManager">Instance of the <see cref="IUserManager"/> interface.</param>
+    /// <param name="networkManager">Instance of the <see cref="INetworkManager"/> interface.</param>
+    /// <param name="httpContextAccessor">Instance of the <see cref="IHttpContextAccessor"/> interface.</param>
+    public DefaultAuthorizationHandler(
+        IUserManager userManager,
+        INetworkManager networkManager,
+        IHttpContextAccessor httpContextAccessor)
+        : base(userManager, networkManager, httpContextAccessor)
+    {
+    }
+
+    /// <inheritdoc />
+    protected override Task HandleRequirementAsync(AuthorizationHandlerContext context, DefaultAuthorizationRequirement requirement)
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="DefaultAuthorizationHandler"/> class.
-        /// </summary>
-        /// <param name="userManager">Instance of the <see cref="IUserManager"/> interface.</param>
-        /// <param name="networkManager">Instance of the <see cref="INetworkManager"/> interface.</param>
-        /// <param name="httpContextAccessor">Instance of the <see cref="IHttpContextAccessor"/> interface.</param>
-        public DefaultAuthorizationHandler(
-            IUserManager userManager,
-            INetworkManager networkManager,
-            IHttpContextAccessor httpContextAccessor)
-            : base(userManager, networkManager, httpContextAccessor)
+        if (ValidateClaims(context.User, out var user))
```

Comment:
```suggestion
        if (ValidateClaims(context.User, out _))
```

Should discard the parameter if it won't be used